lib/git-fastclone.rb

```diff
@@ -356,7 +356,7 @@ def with_git_mirror(url)
             'fatal: missing blob object',
             'fatal: remote did not send all necessary objects',
             /fatal: packed object [a-z0-9]+ \(stored in .*?\) is corrupt/,
-            /fatal: pack has \d+ unresolved deltas/,
+            /fatal: pack has \d+ unresolved delta/,
```

Comment:
I think this change will affect other specs which depend on the pluralization: https://github.com/square/git-fastclone/blob/7d8a6fa118aecf73655634a755fa22e7924c2b72/spec/git_fastclone_runner_spec.rb#L376